util/conditions/v1beta2/getter.go

```diff
@@ -55,6 +55,38 @@ func Get(sourceObj Getter, sourceConditionType string) *metav1.Condition {
 	return meta.FindStatusCondition(sourceObj.GetV1Beta2Conditions(), sourceConditionType)
 }
 
+// Has returns true if a condition with the given type exists.
```

Comment:
Should we have some (pretty basic) test coverage for these utils?

Reply:
Done